src/context/ContextProviders.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { useState } from 'react';
+
+const ContextProviders: React.FC = ({ children }) => {
```

Comment:
En global context der vi skal ha alle contextene samlet etter hver som de lages :) SpråkContext er for seg selv siden SpråkProvider inneholder IntlProvider som må wrappes rundt <App/> for å funke :)